api/src/resource-extraction.js

```diff
@@ -52,21 +56,22 @@ const removeDirectory = () => {
   removeDirectoryRecursive(outputPath);
 };
 
-const extractResources = () => {
-  const extractToDirectory = environment.getExtractedResourcesPath();
+const extractResources = (ddocId, subdir = '') => {
+  const extractToDirectory = path.join(environment.getExtractedResourcesPath(), subdir);
```

Comment:
I'm a little uncomfortable with `admin` being a subdir of the extracted resources path smooshed in there beside all the webapp static resources. Would it be terribly disruptive if we extracted webapp into a `webapp` subdir to keep the two separate?

Reply:
A bit disruptive. 

We also extract default_docs, smooshed in there as well. If we were to separate them, resource extraction would need to differentiate them and extract separately (it doesn't at the moment, all attachments are treated equally). 

We also have to fix path resolution. With admin in the static/admin, paths just match (because all requests to admin static resources are prefixed with /admin). If we put webapp into a subfolder we need to match the paths somehow. 

This is an intermediary step, we won't have resource extraction AT ALL in the next PRs. I don't think it's worth the effort to fix resource extraction just to dump in it 5 seconds. 
If your view is that we should separate webapp from admin (even though static paths match urls perfectly), we should do it in when we start bundling static resources in API, and NOT merge this into `master` right now.

Reply:
My opinion is that we shouldn't separate them, because of the simplicity of paths matching URLs. 
In the current API bundle, all static resources are put into one folder (default_docs _is_ separate). This includes login, static resources, which are in static/login.

Reply:
Yes it's fine for now. Thanks.

Reply:
Thanks! I'll make a note to separate static resources in the bundle, and change this PR to merge into `archv3`.